sdk/identity/identity/README.md

```diff
@@ -179,39 +172,53 @@ const { KeyClient } = require("@azure/keyvault-keys");
 const client = new KeyClient(vaultUrl, credentialChain);
 ```
 
+## Managed Identity Support
+
+The [Managed identity authentication](https://docs.microsoft.com/azure/active-directory/managed-identities-azure-resources/overview) is supported via either the `DefaultAzureCredential` or the `ManagedIdentityCredential` credential classes directly for the following Azure Services:
```

Comment:
> for the following Azure Services

The following are not really "services". Do we want to say "hosted environments"? Feel free to cross check with others in the Identity feature crew to see how they have documented this

Reply:
I like "hosted environments". I found at least one example where these were referred to as "Azure hosts" (when your application is deployed to the following Azure hosts) so this fits.